### PR TITLE
Added HF Model & Space for Classification with GRN and VSN 

### PR DESCRIPTION
**Pull request by shivalikasingh95**
As per the ongoing Hugging Face Keras sprint, I've updated the Classification with GRN and VSN example with links to model and space on HF.
Please let me know if you'd like me to update anything! 

cc: @merveenoyan

### CODE REVIEW

File: examples/structured_data/md/classification_with_grn_and_vsn.md [hunk 1]
```diff
@@ -473,51 +473,75 @@ print(f"Test accuracy: {round(accuracy * 100, 2)}%")
 
 <div class="k-default-codeblock">
 ```
+/home/shivalika/anaconda3/envs/keras/lib/python3.9/site-packages/numpy/core/numeric.py:2463: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
```
Comment (fchollet) [inline examples/structured_data/md/classification_with_grn_and_vsn.md:476]: Please revert all unrelated changes in the `md` file (warnings, etc). Only keep the actual link section that was added.

Comment (shivalikasingh95) [reply examples/structured_data/md/classification_with_grn_and_vsn.md]: Hi @fchollet 
Thanks for pointing out the unrelated changes. I have updated the md file. Please let me know if it's good to take now.

File: examples/structured_data/classification_with_grn_and_vsn.py [hunk 2]
```diff
@@ -96,7 +96,7 @@
     "country_of_birth_self",
     "citizenship",
     "own_business_or_self_employed",
-    "fill_inc_questionnaire_for_veteran's_admin",
+    "fill_inc_questionnaire_for_veterans_admin",
```
Comment (shivalikasingh95) [inline examples/structured_data/classification_with_grn_and_vsn.py:99]: Have added this minor change because while saving the model was getting the following error:
https://github.com/tensorflow/tensorflow/issues/44984

**Review by fchollet — APPROVED** [review]
Thanks for the update!

**Review by fchollet — APPROVED** [review]
Thank you!
